src/demo/ui.jsx

```diff
@@ -90,6 +94,62 @@ const styles = {
     marginRight: '6%',
     marginTop: '2%'
   },
+  confirmationDialogBackground: {
+    backgroundColor: colors.transparentBlack,
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    width: '100%',
+    height: '100%',
+    borderRadius: 10,
+    zPosition: 1
+  },
+  confirmationDialog: {
+    position: 'absolute',
+    backgroundColor: colors.white,
+    color: colors.darkGrey,
+    transform: 'translate(-50%, -50%)',
+    top: '50%',
+    bottom: 'initial',
+    left: '50%',
+    padding: 20
+  },
+  confirmationDialogLeft: {
+    float: 'left',
+    width: '30%'
+  },
+  confirmationDialogRight: {
+    float: 'right',
+    width: '70%'
+  },
+  confirmationHeader: {
+    fontSize: 40,
+    lineHeight: '40px',
```

Comment:
super nit: values like this can be shorthanded to `lineHeight: 40` (like line 126). the only time you _can't_ use that shorthand is for something like lines 141 & 142 where multiple values or a percentage are defined. basically, if you're defining a single value in pixels, you can shorthand to an integer

Reply:
Turns out we always quote the lineHeight values so I fixed all of them.

Reply:
Actually it looks like using a number is converting it to a multiplier of the font size: https://developer.mozilla.org/en-US/docs/Web/CSS/line-height

Reply:
ohhh, interesting -- good catch! TIL 🌈